components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -56,6 +56,14 @@ enum ModelAccess {
   PREMIUM,
 };
 
+enum ModelCapabilities {
+  FAST,
+  SEARCH,
+  THINKING,
+  VISION,
+  TOOLS,
```

Comment:
nit: the values should be cased like `kMyThing`. I've only just learned that they are meant to match C++ style in mojom. But since this is an existing issue we can add to a code health follow-up for this file and references (although this file already has 2 different formats!)

Reply:
Changed to `kMyThing` formatting!